<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AStar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_settings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QPushButton" name="pb_confirm">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Column:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_column">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Row:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_row">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>120</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>MapEdit</string>
     </property>
     <widget class="QRadioButton" name="rb_roadway">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Roadway</string>
      </property>
     </widget>
     <widget class="QSlider" name="hs_penalty">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Penalty</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_obstacle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Obstacle</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_startingposition">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>StartPosition</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_endingposition">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>EndPosition</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_map">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>651</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Map</string>
    </property>
    <widget class="QGraphicsView" name="w_path">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>651</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAStar">
    <property name="title">
     <string>AStar</string>
    </property>
   </widget>
   <addaction name="menuAStar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
